docs/articles/actors/dispatchers.md

```diff
@@ -183,7 +183,10 @@ The `ChannelExectuor` re-uses the same threading settings as the `ForkJoinExecut
 
 ```hocon
 akka.actor.default-dispatcher = {
-    executor = channel-executor
+    executor = channel-executor	
+	# channel executor priority to use
+    # - available: "none", "normal", "idle", "background", "low", "belowNormal", "aboveNormal", "high", "realTime"
```

Comment:
`belownormal` or `below-normal`, `abovenormal` or `above-normal`, `realtime` or `real-time`?

Reply:
there are only 4 priorities (channels), the rest I only included to be equal with windows-process-priority enum

"none" => failure
"idle" or "background" => "idle"
"low" or "belowNormal" => "low"
"normal" => "normal"
"aboveNormal" or  "high" or "realTime" => "high"

Reply:
@Zetanova I want to understand better, is `aboveNormal` `belowNormal` `background` necessary in the light of your comment above?

Reply:
It can be used, but has the same effect as the right side.
In the documentation pls only use "idle", "low", "normal", "high" 
where the default is "normal"

Reply:
Noted. Thanks